README.md

```diff
@@ -9,20 +13,44 @@ A modern approach to Neovim plugin management.
 
 ## :star2: Features
 
-- `Cargo`-like `rocks.toml` file for declaring all your plugins
-- Name-based installation (`use "nvim-neorg/neorg"` becomes `:Rocks install neorg` instead)
-- Automatic dependency and build script management
+- `Cargo`-like `rocks.toml` file for declaring all your plugins.
+- Name-based installation (`use "nvim-neorg/neorg"` becomes `:Rocks install neorg` instead).
+- Automatic dependency and build script management.
 - True semver versioning!
-- (WIP) Automatic running of test suites
+- (WIP) Automatic running of test suites.
+
+## :pencil: Requirements
+
+- An up-to-date `Neovim` nightly (>= 0.10) installation.
+- A C/C++ compiler of your choice (must be compatible with `C++17`!) + [`CMake`](https://cmake.org/).
+- [`rust`](https://www.rust-lang.org/) toolchain (recommended latest stable version).
+- [`luarocks`](https://luarocks.org/) installed on your system and accessible in your shell.
 
-## Installation
+## :hammer: Installation
+
+### :snowflake: Nix
 
 <!-- Nix users have a skill issue here what else can I say.
 Just joking - add an entry for nix users as well. -->
 
+TODO...
```

Comment:
I don't think many Nix users will use a neovim plugin as a plugin manager.
And it's not worth polluting the readme for the few that will (we can use the wiki for that if we need to).